src/scene/scene_scene.js

```diff
@@ -347,13 +372,23 @@ pc.extend(pc, function () {
             material.updateShader = function() {
                 var library = device.getProgramLibrary();
                 var shader = library.getProgram('skybox', {rgbm:scene._skyboxCubeMap.rgbm,
-                    hdr:(scene._skyboxCubeMap.rgbm || scene._skyboxCubeMap.format===pc.PIXELFORMAT_RGBA32F),
-                    fixSeams:scene._skyboxCubeMap.fixCubemapSeams, gamma:scene.gammaCorrection, toneMapping:scene.toneMapping});
+                    hdr: (scene._skyboxCubeMap.rgbm || scene._skyboxCubeMap.format===pc.PIXELFORMAT_RGBA32F),
+                    useIntensity: scene.skyboxIntensity!==1,
+                    mip: scene.skyboxMip,
+                    fixSeams: scene._skyboxCubeMap.fixCubemapSeams, gamma:scene.gammaCorrection, toneMapping:scene.toneMapping});
                 this.setShader(shader);
             };
 
             material.updateShader();
-            material.setParameter("texture_cubeMap", this._skyboxCubeMap);
+            if (!scene._skyboxMip) {
+                material.setParameter("texture_cubeMap", this._skyboxCubeMap);
+            } else {
+                var mip2tex = [null, "64", "16", "8", "4"];
```

Comment:
Is this code right? if scene._skyboxMip is 0 then the code will be var mipTex = this['skyboxPrefilterednull']